coderd/database/queries/provisionerdaemons.sql

```diff
@@ -28,6 +28,54 @@ JOIN
 WHERE
     provisioner_jobs.id = ANY(@provisioner_job_ids :: uuid[]);
 
+-- name: GetProvisionerDaemonsWithStatusByOrganization :many
+SELECT
```

Comment:
This query has a fair bit going on; it might not be a bad idea to add a test in `querier_test.go`. WDYT?

Reply:
I didn't know we had that. What's the purpose of it and the policy for putting something there vs e.g. testing it in the coderd endpoint? I thought I had added the tests but see now that I'm missing tests for the various input parameters.

Reply:
I'm not sure if there's a policy as such, but my intuition is that it's a more convenient and faster place to test various edge cases of queries than doing the whole `coderdtest`.

Reply:
As @johnstcn mentioned, I throw queries there to test more directly and easily when the query is complex. Setting up cases for a full `coderd` test are way more involved and complex then the ones in `queries_test.go`